marlowe/cli-doc/simple-test.sh

```diff
@@ -0,0 +1,283 @@
+                    --tx-out-value 8000000                     \
+                    --tx-out "$ADDRESS_P"+50000000             \
+                    --change-address "$ADDRESS_P"              \
+                    --invalid-before    40000000               \
+                    --invalid-hereafter 80000000               \
+                    --out-file tx.raw                          \
+| sed -e 's/^TxId "\(.*\)"$/\1/'
+)
+
+marlowe-cli submit $MAGIC_FLAG $MAGIC_NUM                    \
+                   --socket-path "$CARDANO_NODE_SOCKET_PATH" \
+                   --required-signer $PAYMENT_SKEY           \
+                   --tx-body-file tx.raw
+
+
+# Wait until the transaction is appears on the blockchain.
+
+timeout 10m bash << EOI
+until (cardano-cli query utxo $MAGIC_FLAG $MAGIC_NUM --address "$ADDRESS_S" | grep "$TX_3" > /dev/null)
+do
+  sleep 5s
+done
+EOI
+
+cardano-cli query utxo $MAGIC_FLAG $MAGIC_NUM --address "$ADDRESS_S"
+
+
+# 3. Withdrawn the remaining 8 ADA.
+
+echo -e \\nWITHDRAW THE REMAINING 8 ADA\\n
+
+TX_4=$(
+marlowe-cli close $MAGIC_FLAG $MAGIC_NUM                    \
+                  --socket-path "$CARDANO_NODE_SOCKET_PATH" \
+                  --tx-in-script-file example.plutus        \
+                  --tx-in-redeemer-file example-0.redeemer  \
+                  --tx-in-datum-file example-0.datum        \
+                  --tx-in-marlowe "$TX_3"#1                 \
+                  --tx-in "$TX_3"#0                         \
+                  --tx-in-collateral "$TX_3"#0              \
+                  --tx-out "$ADDRESS_P"+8000000             \
+                  --change-address "$ADDRESS_P"             \
+                  --invalid-before    40000000              \
+                  --invalid-hereafter 80000000              \
+                  --out-file tx.raw                         \
+| sed -e 's/^TxId "\(.*\)"$/\1/'
+)
+
+marlowe-cli submit $MAGIC_FLAG $MAGIC_NUM                    \
+                   --socket-path "$CARDANO_NODE_SOCKET_PATH" \
+                   --required-signer $PAYMENT_SKEY           \
+                   --tx-body-file tx.raw
+
+
+# See that the transaction succeeded.
+
+timeout 10m bash << EOI
+until (cardano-cli query utxo $MAGIC_FLAG $MAGIC_NUM --address "$ADDRESS_P" | grep "$TX_4" > /dev/null)
+do
+  sleep 5s
+done
+EOI
+
+cardano-cli query utxo $MAGIC_FLAG $MAGIC_NUM --address "$ADDRESS_S"
+
+cardano-cli query utxo $MAGIC_FLAG $MAGIC_NUM --address "$ADDRESS_P"
+
+echo -e \\nSUCCESS\\n
+
+
+# Clean up.
+
+echo -e \\nCLEAN UP\\n
+
+cardano-cli query utxo $MAGIC_FLAG $MAGIC_NUM --address "$ADDRESS_P" --out-file /dev/stdout \
+| jq '. | to_entries[] | .key'                                                              \
+| sed -e 's/"//g;s/^/--tx-in /'                                                             \
+| xargs cardano-cli transaction build --alonzo-era $MAGIC_FLAG $MAGIC_NUM                   \
```

Comment:
I think this `xargs` call is unsafe and needs to make allowance for there being many `--tx-in`s.

Reply:
Fixed in e6ccd956be3dea9eb9181d6f3737cd297288943e.